app/src/main/java/org/fossasia/susi/ai/signup/SignUpActivity.kt

```diff
@@ -212,7 +212,9 @@ class SignUpActivity : AppCompatActivity(), ISignUpView {
 
     private fun signUpToTermsConditionPage() {
         signUpToTermsCondition.setOnClickListener {
-            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse("https://chat.susi.ai/privacy")))
+            val intent = Intent(this@SignUpActivity, SkillsActivity::class.java)
+            intent.putExtra("fromWhere", 1)
```

Comment:
What is one?

Reply:
Sir, I have used one as a parameter value to tell the SkillsActivity that the call is from SignUpActivity and it should open Privacy Fragment instead of SkillsFragment. I used it as a techniques to open fragment of an activity from another activity.

Reply:
Use proper naming conventions and extract in a constant